README-zh_CN.md

```diff
@@ -52,7 +52,11 @@ Lunettes 可以识别容器生命周期每个交付阶段的开始和结尾，
 ```bash
 # install lunettes
 # 该过程开启apiserver的审计功能，apiserver会发生重启
-helm upgrade --install lunettes oci://registry-1.docker.io/lunettes/lunettes-chart --version [version] 
+helm upgrade --install lunettes oci://registry-1.docker.io/lunettes/lunettes-chart --version [version]
+
+#如果kindest/node版本 < v1.21，需要设置policyVersion的版本
```

Comment:
What should be emphasized here is the kubernetes version rather than the kind version

Reply:
done